node/core/approval-voting/src/lib.rs

```diff
@@ -791,6 +797,7 @@ where
 			&mut approvals_cache,
 			&mut subsystem.mode,
 			actions,
+			if counter.fetch_add(1, Ordering::SeqCst) == offset { true } else { false },
```

Comment:
the if case can be dropped :) just use the boolean expression

Reply:
I added logging in these two cases, so for now I'll leave it as an if-statement. I'll keep the logging for deploying to one node but can drop the logging before deploying to all validators.